ui/tests/acceptance/token-test.js

```diff
@@ -317,7 +315,8 @@ module('Acceptance | tokens', function (hooks) {
         .dom('.flash-message.alert-error')
         .doesNotExist('No notification yet for a token with 10m5s left');
       notificationNotRendered();
-      setTimeout(() => {
+      setTimeout(async () => {
+        await percySnapshot(assert);
```

Comment:
`thought`: Whoa! This is super interesting! Why did we do this?

Reply:
Thanks, I like this one a lot. We're testing that a soon-to-expire token generates an alert for the user.

So, we create a token that expires in 10min5sec, and check to make sure that there's no message
Then we wait 5sec, and check to make sure there IS a message. We take a screenshot for visual diffs.